Source/WebCore/animation/TimelineRange.cpp

```diff
@@ -150,28 +150,26 @@ SingleTimelineRange SingleTimelineRange::parse(TimelineRangeValue&& value, RefPt
         CSSTokenizer tokenizer(rangeString);
         auto tokenRange = tokenizer.tokenRange();
         tokenRange.consumeWhitespace();
-        if (auto consumedRange = CSSPropertyParserHelpers::consumeAnimationRange(tokenRange, parserContext, type))
-            return range(*consumedRange, type, nullptr, element);
-        return SingleTimelineRange { };
+        return CSSPropertyParserHelpers::consumeAnimationRange(tokenRange, parserContext, type);
     },
     [&](TimelineRangeOffset& rangeOffset) {
         CSSTokenizer tokenizer(rangeOffset.rangeName);
         auto tokenRange = tokenizer.tokenRange();
         tokenRange.consumeWhitespace();
         if (auto consumedRangeName = CSSPropertyParserHelpers::consumeAnimationRange(tokenRange, parserContext, type)) {
             if (rangeOffset.offset)
-                return range(CSSValuePair::createNoncoalescing(*consumedRangeName, *rangeOffset.offset->toCSSValue()), type, nullptr, element);
-            return range(*consumedRangeName, type, nullptr, element);
+                return RefPtr<CSSValue> { CSSValuePair::createNoncoalescing(*consumedRangeName, *rangeOffset.offset->toCSSValue()) };
+            return consumedRangeName;
         }
         if (RefPtr offset = rangeOffset.offset)
-            return range(*offset->toCSSValue(), type, nullptr, element);
-        return SingleTimelineRange { };
+            return offset->toCSSValue();
+        return RefPtr<CSSValue> { nullptr };
```

Comment:
Surely just `return nullptr` would work here with the type specified as the lambda's return vale.

Reply:
Fixed.